src/arch/InputHandler/InputHandler_DirectInput.cpp

```diff
@@ -517,11 +501,8 @@ InputHandler_DInput::UpdatePolled(
 								ButtonPressed(DeviceInput(dev, pos, 0, tm));
 							}
 						} else
-							LOG->MapLog("unknown input",
-										"Mouse '%s' is returning an unknown "
-										"mouse offset, %i",
-										device.m_sName.c_str(),
-										in.ofs);
+							Locator::getLogger()->trace("Mouse '{}' is returning an unknown mouse offset, {}",
```

Comment:
We want to be careful converting MapLogs to traces due to their possibly high traffic usage. 
In this specific case, it might be better to make this a warn. If it crops up as an issue later, we could remove this log entirely or find a new solution.

Reply:
Fixed.